### ISSUE TITLE
Distortion when resizing window 

### ISSUE THREAD

**Issue by dhthwy** [rlt3/SDL-Console#2]
For some reason the projection matrix isn't updating? I'm not sure as I'm an opengl newbie.

In _Console_set_window_size
I was able to fix this behavior with:

```
glViewport(0, 0, r, t);
glUseProgram(tty->shader_prog);

GLfloat orthoMatrix[4*4] = {
        2.0f / (r - l),   0.0f,             0.0f, 0.0f,
        0.0f,             2.0f / (t - b),   0.0f, 0.0f,
        0.0f,             0.0f,            -1.0f, 0.0f,
        -(r + l)/(r - l), -(t + b)/(t - b), 0.0f, 1.0f,
    }; 

glUniformMatrix4fv(glGetUniformLocation(tty->shader_prog, "projection"),
           1, GL_FALSE, orthoMatrix);
```
I'm confused by the need to call glUseProgram again, but for me, it was necessary to call when resizing. 

*EDIT* nm, I see the program being unset now.

**Comment by wayfarerdb** [comment]
Did you figure it out?

**Comment by dhthwy** [comment]
I believe so. PR submitted if you want the changes.

**Comment by wayfarerdb** [comment]
fixed with #3
